code/modules/admin/tickets/admintickets.dm

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
The entire spacing of this entire file is incorrect.

Reply:
Yeah Tigercat told me this also. Because they were new files Atom was using soft tabs instead of hard tabs. I've fixed it for the next commit where I'm also addressing some of Tiger's feedback.

Reply:
This was addressed in the latest commit.